src/main/mdo/assembly.mdo

```diff
@@ -226,7 +226,7 @@
         </field>
         <field>
           <name>repositories</name>
```

Comment:
Can we remove also `repositories` from model as we remove it from code?

Reply:
Ah, did I miss range from class? As field is gone, that's sure

Reply:
removed